translate/v35tov34/v35tov34.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package v35tov34
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/coreos/ignition/v2/config/translate"
+	"github.com/coreos/ignition/v2/config/v3_4/types"
+
+	old_types "github.com/coreos/ignition/v2/config/v3_5/types"
+	"github.com/coreos/ignition/v2/config/validate"
+)
+
+// Copy of github.com/coreos/ignition/v2/config/v3_4/translate/translate.go
```

Comment:
Comments are outdated need to be updated to reflect this new version

Reply:
to that fact, did you copy the ignition translate block for 3_5? or is this a copy from the old convert file 3_4>3_3 ?

Reply:
I copy the transalte block for 3_5, but I copied the comments for the other file and forgot to change it. Also, I had to change the name of the last copied method. Is that a problem? 

Here we have the original: https://github.com/coreos/ignition/blob/main/config/v3_5/translate/translate.go